src/index.html

```diff
@@ -1,14 +1,29 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="all">
+    <header>
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li><a class="is-active" href="#">Apple</a></li>
+          <li><a  href="#">Samsung</a></li>
+          <li><a  href="#">Smartphones</a></li>
+          <li><a href="#" data-qa="hover">Laptops & computers</a></li>
+          <li><a  href="#">Gadgets</a></li>
+          <li><a  href="#">Tablets</a></li>
+          <li><a  href="#">Photo</a></li>
+          <li><a  href="#">Video</a></li>
```

Comment:
to access most of the elements we are using classes most of the time - they are way easier to work with than using complicated CSS selectors, please provide some kind of class on `<li>`, `<a>`